content/2024/chinese/speakers.zh.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "演讲人"
+seo_title: "JakartaOne Livestream - Chinese | Jakarta EE Software | Cloud Native"
```

Comment:
The `seo_title` here should also be unique.

Reply:
Done.